<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Segoe Print&quot;;background-color:</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Text : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="SifrujBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(255, 42, 23)}</string>
    </property>
    <property name="text">
     <string>Šifruj</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>hodnota a :</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>hodnota b :</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vstup_text">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(141, 255, 133)}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DesifrujBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(255, 42, 23)}</string>
    </property>
    <property name="text">
     <string>Dešifruj</string>
    </property>
   </widget>
   <widget class="QLabel" name="sifra_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Šifrovaný text :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;ABCDEFGHIJKLMNOPQRSTUVWXYZ&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vstup_text_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vstupný text :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_text">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(141, 255, 133)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="a_entry">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>89</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(141, 255, 133)}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b_entry">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(141, 255, 133)}</string>
    </property>
   </widget>
   <widget class="QLabel" name="desifra_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dešifrovaný text :</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="desifra_text">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser{background-color:rgb(255, 225, 149)}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:rgb(255, 42, 23)}</string>
    </property>
    <property name="text">
     <string>Vymaž</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifra_text">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:rgb(141, 255, 133)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
